app/views/handlers/question.py

```diff
@@ -1,8 +1,15 @@
+from app.questionnaire.placeholder_renderer import PlaceholderRenderer
+from app.questionnaire.questionnaire_store_updater import QuestionnaireStoreUpdater
+from app.questionnaire.schema_utils import transform_variants
 from app.views.handlers.block import BlockHandler
 from app.views.contexts.question import build_question_context
 
 
 class Question(BlockHandler):
+    @property
```

Comment:
Should this (and others) be a cached property?

Reply:
Good point - it's an easy optimisation

Reply:
I wasn't expecting this to be applied to questionnaire_store_updater too, since it's already cached through that if statement, but I guess it doesn't really make much difference.